examples/travel-lb.amp.html

```diff
@@ -0,0 +1,197 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Travel date picker example</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-custom>
+  amp-lightbox {
+    background: rgba(0, 0, 0, .75);
+  }
+
+  .date-picker-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+  }
+
+  .button-area {
+    display: block;
+    position: fixed;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    margin: auto;
+    z-index: 10;
+    height: 40px;
+
```

Comment:
nit: extra space?

Reply:
Sometimes I like to use an empty line to separate positioning- and layout-related properties like `display`, `margin`, `padding`, `width`, etc vs presentation-related properties, but here that doesn't matter too much so I can eliminate it